frontend/src/metabase/admin/settings/components/CloudPanel/MigrationStart.tsx

```diff
@@ -20,27 +17,9 @@ export const MigrationStart = ({
   const [isModalOpen, { turnOn: openModal, turnOff: closeModal }] =
     useToggle(false);
 
-  const plan = useSelector(state =>
-    getPlan(getSetting(state, "token-features")),
-  );
-  const isProSelfHosted = plan === "pro-self-hosted";
-
   return (
     <>
-      <Box mt="1rem">
-        <Text size="md">
-          {isProSelfHosted
-            ? t`Migrate this instance to Metabase Cloud at no extra cost and get high availability, automatic upgrades, backups, and enterprise grade compliance.`
-            : t`Migrate this instance to Metabase Cloud with a free 14-day trial and get high availability, automatic upgrades, backups, and official support.`}{" "}
-          <ExternalLink href="https://www.metabase.com/cloud/">{t`Learn More.`}</ExternalLink>
-        </Text>
-
-        <Button
-          mt="2rem"
-          variant="filled"
-          onClick={openModal}
-        >{t`Get started`}</Button>
-      </Box>
+      <UpsellCloud onOpenModal={openModal} source="settings-cloud" />
```

Comment:
This is the key change when it comes to the `/admin/settings/cloud` UI.
We're replacing the old custom CTA with the component from specifically designed within the broader Upsell system.